code/modules/cm_aliens/structures/trap.dm

```diff
@@ -347,6 +348,11 @@
 	if(ismob(A) || isVehicleMultitile(A))
 		HasProximity(A)
 
+/obj/effect/alien/resin/trap/proc/on_shuttle_crushing()
+	SIGNAL_HANDLER
+	UnregisterSignal(src, COMSIG_MOVABLE_SHUTTLE_CRUSH)
+	loc = null
```

Comment:
```suggestion
	moveToNullspace()
```

Reply:
Moved the force triggering outside of Destroy() and put it into healthcheck(), which should handle qdelling. Destroy() now only handles clearing up references. (I hope??)

Tested it again, this makes traps not trigger upon getting shuttlecrushed, but they trigger when they get damaged.